.github/workflows/trunk.yml

```diff
@@ -86,7 +86,7 @@ jobs:
       build-environment: libtorch-linux-bionic-cuda11.7-py3.7-gcc7
       docker-image-name: pytorch-linux-bionic-cuda11.7-cudnn8-py3-gcc7
       build-generates-artifacts: false
-      runner: linux.4xlarge
+      runner: linux.12xlarge  # 4xlarge runs out of memory after https://github.com/pytorch/pytorch/pull/88575
```

Comment:
Are we entirely sure that we'll have enough capacity for this? 12 core machines on every trunk job seems like it'd be lot

Reply:
One PR is affecting this this badly?

Is this a revert-worthy performance regression?

Reply:
It looks like the issue has recoverd after 4 consecutive failures.  Now it's running fine.  Because this is a build job, this makes me think that something has been built and cached, thus avoiding OOM.  Let me try to do a quick test without caching to see how it fares.  Normally, we can just slow down the build the bit by setting a lower `MAX_JOBS`.  If that's the case, I will retrofit this PR to do that.

Reply:
Going to reduce `MAX_JOBS` when building libtorch from the default of 14 `$(nproc --ignore=2)` to 12 `$(nproc --ignore=2)` to see if that helps